Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.7-alpine
+
+RUN apk add --no-cache --virtual build-dependencies gcc musl-dev
+RUN pip3 install --no-cache-dir homeassistant-cli==0.4.4
+RUN apk del build-dependencies
```

Comment:
setup a entrypoint or cmd so you don't need to add `hass-cli` as first argument.

Reply:
i had it this way in the first version but then thought this could be unnecessary confusing for some user (which maybe do not work with docker every day...)

Reply:
I say add it back as you can easily override it and it makes it cleaner imo. we'll just doc using an alias anyway :)